contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1336,13 +1336,13 @@ define_escape_hatch_variables(void)
 
 	/* fulltext */
 	DefineCustomEnumVariable("babelfishpg_tsql.escape_hatch_fulltext",
-							 gettext_noop("escape hatch for fulltext"),
+							 gettext_noop("escape hatch for fulltext search"),
 							 NULL,
 							 &escape_hatch_fulltext,
 							 EH_STRICT,
 							 escape_hatch_options,
-							 PGC_USERSET,
-							 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
+							 PGC_SUSET,
```

Comment:
If a normal user tries to use this escape hatch to turn on fulltext search, it will throw `permission error`. This can only be turned on if the login is of super user. We have just put it like this so that let's say we roll FTS feature in prod, the customer shouldn't be able to use it rather only we, as superuser, should be able to turn it on. We'll rollback these extra changes of escape hatch once the pentest is done, that's what we have planned.